modules/install/pages/install-production-deployment.adoc

```diff
@@ -58,3 +58,16 @@ However, you might find a need to have a smaller deployment for test and develop
 
 xref:deployment-considerations-lt-3nodes.adoc[About Deploying Clusters With Less Than Three Nodes]
 |===
+
+== General Guidelines
+
+Use RAID 1+0 (or RAID 1) if Couchbase bucket replication is 1 or lower:: If bucket replicas are set to 2 or higher, the administrator might rely on database-level replication for data durability.
+
+Linux OS::
+* When deploying Couchbase Server on production Linux, you should use either the XFS or ext4 file system.
+XFS is preferred because it provides better performance when multiple IO streams are accessing the file system, such as during Data Service compaction or when the working set exceeds the memory quota and the data is read from disk.
```

Comment:
"performance" is a somewhat vague term, and our users might pull us up on it. I think we can be more specific here - something like:

> "XFS is preferred because it provides more consistent latency (fairer scheduling) when multiple IO streams are accessing the file system, such as during Data Service compaction or when the working set exceeds the memory quota and the data is read from disk.

Reply:
Updated.